ext/nnstreamer/include/nnstreamer.proto

```diff
@@ -32,6 +32,12 @@ message Tensors {
   }
   frame_rate fr = 2;
   repeated Tensor tensor = 3;
+  enum Tensor_format {
+    NNS_TENSOR_FORAMT_STATIC = 0;
+    NNS_TENSOR_FORMAT_FLEXIBLE = 1;
+    NNS_TENSOR_FORMAT_SPARSE = 2;
+  }
+  Tensor_format format = 4;
```

Comment:
default static? (format = NNS_TENSOR_FORAMT_STATIC;)

Reply:
For enums, the default value is the first value listed in the enum's type definition.
So, `NNS_TENSOR_FORAMT_STATIC` is default format value.
`Tensor_format format = 4;`, this means field number is 4.